Misc/NEWS.d/next/Library/2023-11-18-16-30-21.gh-issue-112240.YXS0tj.rst

```diff
@@ -0,0 +1 @@
+Add option to calendar module CLI to specify the weekday to start each week
```

Comment:
```suggestion
Add option to calendar module CLI to specify the weekday to start each week.
```

(Feel free to also append "Patch by Steven Ward.")

Reply:
I applied your changes.